src/components/FeaturedArticleSmall.jsx

```diff
@@ -0,0 +1,32 @@
+export default function ArticlesList({ articles }) {
+
+  if (!articles) {
+    return <h3>Loading...</h3>;
+  }
+
+  return (
+    <div className="row mb-2">
```

Comment:
If this is a list, could you use a ul and li tags to add to the semantics for screen readers?